manifests/i/ITK-SNAP/ITK-SNAP/4.0.2/ITK-SNAP.ITK-SNAP.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ITK-SNAP.ITK-SNAP
+PackageVersion: 4.0.2
+InstallerType: nullsoft

```

Comment:
```suggestion
InstallerType: nullsoft
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Ah, I tried installing the manifest and running the software on my local machine and installing it in the sandbox, but I forgot to run it in the sandbox so I didn't find this dependency. Thanks for testing!